motor/frameworks/asyncio/__init__.py

```diff
@@ -66,6 +66,19 @@ def get_future(loop):
 _EXECUTOR = ThreadPoolExecutor(max_workers=max_workers)
 
 
+def _reset_global_executor():
+    """Re-initialize the global ThreadPoolExecutor"""
+    global _EXECUTOR
+    _EXECUTOR = ThreadPoolExecutor(max_workers=max_workers)
+
+
+if hasattr(os, "register_at_fork"):
+    # This will run in the same thread as the fork was called.
+    # If we fork in a critical region on the same thread, it should break.
+    # This is fine since we would never call fork directly from a critical region.
```

Comment:
What is this comment trying to say?

Reply:
I copied this over from Ben's PR, which had this comment for all of the register_at_fork calls. I can remove if you think it is not valuable?

Reply:
Yeah I don't think its valuable. The problem is that it's only describing _what_ after_in_child does. If there was a comment here it should explain _why_ we call it.

Reply:
Should be fixed now.